packages/camera/camera_android_camerax/pigeons/camerax_library.dart

```diff
@@ -417,3 +419,14 @@ abstract class FallbackStrategyHostApi {
   void create(int identifier, VideoQuality quality,
       VideoResolutionFallbackRule fallbackRule);
 }
+
+@HostApi(dartHostTestHandler: 'TestCameraControlHostApi')
+abstract class CameraControlHostApi {
+  @async
+  void enableTorch(int identifier, bool torch);
```

Comment:
@bparrishMines If the `Result` this returns ends up with an error, would I be able to catch this from the Dart side and throw an error there?

Reply:
Yea, an exception on the Java side should cause a `PlatformException` error on the Dart side.

Reply:
I'm going to modify this PR to catch `PlatformException`s that may be cause by CameraX failing to turn on torch mode.